omod/src/main/compass/sass/allergy.scss

```diff
@@ -15,14 +20,15 @@
   }
 
   .inline {
-      display: inline;
```

Comment:
It is always best to have formatting changes in a pull request of their own. That way, this one get only those changes required to fix the reported problem, hence simplifying the reviewer's work.